readium/navigator/src/main/java/org/readium/r2/navigator/epub/EpubNavigatorFactory.kt

```diff
@@ -47,6 +52,9 @@ class EpubNavigatorFactory(
             publication = publication,
             baseUrl = null,
             initialLocator = initialLocator,
+            readingOrder = readingOrder ?: publication.readingOrder,
+            positionsByReadingOrder = if (readingOrder == null)
+                Collections.emptyList() else runBlocking { publication.positionsByReadingOrder() },
```

Comment:
This is no longer needed.

Reply:
Thanks, removed now. (Starting to get a bit embarrassed now over my Kotlin short-comings)..